packages/app-mobile/services/e2ee/crypto.ts

```diff
@@ -48,4 +46,4 @@ const Krypto: KryptoInterface = {
 	},
 };
 
-export default Krypto;
+export default cryptoLib;
```

Comment:
I suggested naming it cryptoLib here to avoid a conflict with "crypto" from "react-native-quick-crypto", but everywhere else it should be named just "crypto". Because that's the name of the file (see coding style), so please rename it everywhere to just "crypto" (except here).

Also please read the coding style carefully especially as it relates to naming conventions, as that will save you a lot of time in reviews: https://joplinapp.org/help/dev/coding_style

Reply:
I guess I got you point.
The default export name of `react-native-quick-crypto` is actually `QuickCrypto`. For the consistency and avoiding conflicts I've changed the import name of `react-native-quick-crypto` to `QuickCrypto`. Then I changed all `cryptoLib` to `crypto` for the consistency.

I also read the coding style and found the imports in `packages/lib/services/e2ee/crypto.ts` don't meet the requirement, so I made the following changes:
- Remove the `required` style of imports
- Only import used functions
- Rename the imported functions to prevent conflicts